libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/Queue/CSFNetwork+Salesforce.m

```diff
@@ -38,20 +40,53 @@ - (void)setDefaultConnectCommunityId:(NSString *)defaultConnectCommunityId {
 
 - (void)setupSalesforceObserver {
     [[NSNotificationCenter defaultCenter] addObserver:self
-                                             selector:@selector(userAccountManagerDidChangeCurrentUser:)
-                                                 name:SFUserAccountManagerDidChangeCurrentUserNotification
+                                             selector:@selector(userAccountManagerDidChangeUserData:)
+                                                 name:SFUserAccountManagerDidChangeUserDataNotification
+                                               object:nil];
+
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(userAccountManagerDidChangeUser:)
+                                                 name:SFUserAccountManagerDidChangeUserNotification
                                                object:nil];
 }
 
 #pragma mark SFAuthenticationManagerDelegate
+- (void)userAccountManagerDidChangeUser:(NSNotification*)notification {
```

Comment:
So let's ditch this notification handling entirely.  It's superfluous now that we have the two notification types.